src/service/receipt.service.ts

```diff
@@ -0,0 +1,79 @@
+
+import {
+    ArtifactExpiryReceipt,
+    BalanceChangeReceipt,
+    BalanceTransferReceipt,
+    InflationReceipt,
+} from 'nem2-sdk';
+
+export class ReceiptService {
+    constructor() {
+
+    }
+
+    /**
+     * @description The logic of formatting Receipt info
+     * @param receipt
+     * @returns {string}
+     */
+    public formatReceiptToFilter(statement: any): string {
+        let txt = '';
+        if (statement.transactionStatements[0]) {
+            const receiptItem = statement.transactionStatements[0].receipts[0];
```

Comment:
One block has multiple transaction statements and each transaction statement has more than one receipt. Right now you are rendering only one receipt from the first transaction statement.